packages/salesforcedx-vscode-apex/test/vscode-integration/views/testOutlineNamespace.test.ts

```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
```

Comment:
Very similar to the tests in `testOutline.test.ts` but running with data that has a namespace.